kafka-handler/pom.xml

```diff
@@ -114,8 +114,21 @@
       <version>1.7.25</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.confluent</groupId>
+      <artifactId>kafka-streams-avro-serde</artifactId>
```

Comment:
You're not using Kafka Streams or the serde.

Maybe you want `kafka-avro-serializer`? Or the schema registry client like you had before?

Reply:
Yes, I'm using it in `AvroBytesConverterTest` class and it is `KafkaAvroSerializer avroSerializer = new KafkaAvroSerializer(new MockSchemaRegistryClient());`

Reply:
That's just a serializer, not a Kafka **Streams** serde.

You can relax the dependency to `kafka-avro-serializer` instead here. That's what I'm saying

Reply:
Bump?